lib/run.js

```diff
@@ -10,7 +10,7 @@ const { findPrivateKey } = require('probot/lib/private-key');
 const { createProbot } = require('probot');
 const app = require('./configure-robot');
 
-const workers = process.env.WEB_CONCURRENCY || 1;
+const workers = Number(process.env.WEB_CONCURRENCY || 1);
```

Comment:
This worked before?

Reply:
Yes! `throng` will cast it to a `Number` if we pass in a string, but since I wanted to do the comparison at the end of this file, I needed it to be a `Number` ahead of passing it to `throng`.